doc/howtos/building-images.rst

```diff
@@ -72,17 +101,18 @@ For currently :ref:`platforms`, the appropriate ``MACHINE`` selections are:
     BeagleBone Black            beaglebone
     ==========================  ====================================
 
-Virtual machine images (a :file:`.vdi` file) are created for each of these builds hardware platforms as part 
-of the build process (and included in the prebuilt image folders too).
+Virtual machine images (a :file:`.vdi` file) are created for the ``intel-corei7-64``  hardware platforms as part 
+of the build process (and included in the prebuilt image folder too).
 
 Base Images
 -----------
 
-``ostro-image.bb`` is the image recipe used by the Ostro
-project. It uses image features (configured via ``IMAGE_FEATURES``) to
+In your cloned ``ostro-os`` repository folder, ``./meta-ostro/recipes-image/images/ostro-image.bb`` is the 
+image recipe used for the Ostro
+project. This recipe file uses image features (configured via ``IMAGE_FEATURES``) to
```

Comment:
I think it would also be appropriate to mention that we recommend users to tweak the content of their image using OSTRO_EXTRA_IMAGE_FEATURES (set in their conf/local.conf file and documented there via #79).